api/nodes/serializers.py

```diff
@@ -1497,6 +1497,32 @@ class DraftRegistrationSerializer(JSONAPISerializer):
     def get_absolute_url(self, obj):
         return obj.absolute_url
 
+    def update_metadata(self, draft, metadata, reviewer, required_fields=False):
```

Comment:
Consolidates this because it was being called 2x - once on create, and once on updating a DraftRegistration